repo/packages/covfie/package.py

```diff
@@ -0,0 +1,29 @@
+from spack.package import *
+
+
+class Covfie(CMakePackage):
+    homepage = "https://github.com/acts-project/covfie"
+    git = "https://github.com/acts-project/covfie.git"
+    list_url = "https://github.com/acts-project/covfie/tags"
+
+    version("0.5.0", commit="59bd2e0a4861db6e5898606e0167da895505e46b")
+
+    variant("cuda", default=False, description="Enables the CUDA platform")
```

Comment:
If you want this to be usable for covfie development, consider adding tests, examples, benchmarks too.